libnavigation-ui/src/main/java/com/mapbox/navigation/ui/NavigationViewModel.java

```diff
@@ -42,8 +42,9 @@
 import com.mapbox.navigation.ui.camera.DynamicCamera;
 import com.mapbox.navigation.ui.feedback.FeedbackItem;
 import com.mapbox.navigation.ui.instruction.BannerInstructionModel;
-import com.mapbox.navigation.ui.instruction.InstructionModel;
-import com.mapbox.navigation.ui.junction.RouteJunctionModel;
+import com.mapbox.navigation.ui.instruction.internal.InstructionModel;
```

Comment:
Typo, it should be `com.mapbox.navigation.ui.internal.instruction.InstructionModel`